.rubocop_todo.yml

```diff
@@ -1,19 +1,30 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude`
-# on 2021-06-23 14:46:36 UTC using RuboCop version 0.93.1.
+# on 2021-08-02 18:08:55 UTC using RuboCop version 0.93.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 3
+Lint/MissingSuper:
+  Exclude:
+    - 'lib/psulib_traject/processors/call_number/dewey.rb'
+    - 'lib/psulib_traject/processors/call_number/lc.rb'
+    - 'lib/psulib_traject/processors/call_number/other.rb'
+
+# Offense count: 1
+Lint/MixedRegexpCaptureTypes:
+  Exclude:
+    - 'lib/psulib_traject/processors/call_number/lc.rb'
+
```

Comment:
@awead what do you think for moving these (and some of the other too) to rubocop.yml rather than todo?

Reply:
I feel like we will just move forward with these exclusions 🤔